docs/deploy-hyperlane.mdx

```diff
@@ -128,10 +128,11 @@ To send a test message, run:
 ```bash
 hyperlane send message --relay
 ```
-
+:::info
 Currently core self-relay only works **from Holesky** **to your custom chain**. When dispatch is successful, you should see that the message was self-relayed!
```

Comment:
@ltyu - I was able to self relay from Sepolia, is it supoosed to only be from Holesky?

Reply:
this was patched but its a bit hacky
https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/c2c5bb9bbcbb732f6697caa4f81bb94a69f523fb/typescript/cli/src/status/message.ts#L90

Reply:
but yes other origins should work now